examples/apps/attributable-map/src/modelContainerRuntimeFactoryWithAttribution.ts

```diff
@@ -0,0 +1,93 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import {
+	IContainer,
+	IContainerContext,
+	IRuntime,
+	IRuntimeFactory,
+} from "@fluidframework/container-definitions";
+import { IContainerRuntimeOptions, ContainerRuntime } from "@fluidframework/container-runtime";
+import { IContainerRuntime } from "@fluidframework/container-runtime-definitions";
+import { NamedFluidDataStoreRegistryEntries } from "@fluidframework/runtime-definitions";
+import {
+	mixinAttributor,
+	createRuntimeAttributor,
+	IProvideRuntimeAttributor,
+} from "@fluid-experimental/attributor";
+import { FluidObject } from "@fluidframework/core-interfaces";
+import { makeModelRequestHandler } from "@fluid-example/example-utils";
+
+const containerRuntimeWithAttribution = mixinAttributor(ContainerRuntime);
```

Comment:
TBH I'm not a fan of the pseudo-mixin pattern that gets used in various places in the repo (not just for attribution).  Doesn't need to block this example, but maybe consider whether there are better options for how to introduce attribution.